### ISSUE TITLE
Creating a 1-1 transaction works fine with one input but fails on multiple inputs

### ISSUE THREAD

**Issue by abhinavlanka** [bitcoinjs/bitcoinjs-lib#1054]
When I construct a transaction with one input, it works fine but when I try constructing a transaction with multiple inputs, I get the following error. Has anyone else encountered this error? 
```
transaction_builder.js:632 Uncaught Error: Transaction is not complete
at TransactionBuilder../node_modules/bitcoinjs-lib/src/transaction_builder.js.TransactionBuilder.__build
at TransactionBuilder../node_modules/bitcoinjs-lib/src/transaction_builder.js.TransactionBuilder.build
```

**Comment by dcousens** [comment]
@abhinavlanka `Transaction is not complete`,  in Bitcoin layman terms,  means "not enough signatures" (or in this case, missing scriptSig data).

It is the verbatim term for that issue.
